messaging-modules/rabbitmq/src/test/java/com/baeldung/benchmark/queue/dynamic/DynamicQueueCreationLiveTest.java

```diff
@@ -0,0 +1,78 @@
+package com.baeldung.benchmark.queue.dynamic;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+
+public class DynamicQueueCreationLiveTest {
+
+    private static final String QUEUE_NAME = "baeldung-queue";
+    private static final String QUEUE_NAME_NEW = "baeldung-queue-new";
+
+    private static Connection connection;
+    private static Channel channel;
+
+    @BeforeAll
+    public static void setUpConnection() throws IOException, TimeoutException {
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setHost("localhost");
+        connection = factory.newConnection();
+    }
+
+    @BeforeEach
+    public static void setUpChannel() throws IOException {
+        channel = connection.createChannel();
+    }
```

Comment:
The pre and post method for every test should not be static. If you need a method before the set of test you should use @BeforeAll.